x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/types.ts

```diff
@@ -87,8 +103,10 @@ export interface AboutStepRuleDetails {
   description: string;
 }
 
+export type SeverityValue = 'low' | 'medium' | 'high' | 'critical';
+
 export interface AboutStepSeverity {
-  value: string;
+  value: SeverityValue;
```

Comment:
Last one on the `SeverityValue` promise... 😅 , but would it be possible to start using the `common` types here?

e.g.

https://github.com/spong/kibana/blob/d2156b7285c0397fe93a12cd8f1464ac430bc0e8/x-pack/plugins/security_solution/common/detection_engine/schemas/common/schemas.ts#L256-L257